addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/config/EnOceanChannelDimmerConfig.java

```diff
@@ -20,8 +20,12 @@
 public class EnOceanChannelDimmerConfig {
 
     public Integer rampingTime;
+    public Boolean eltakoDimmer;
+    public Boolean storeValue;
 
     public EnOceanChannelDimmerConfig() {
         rampingTime = 0;
```

Comment:
Please assign default values directly so that you can remove the constructor.

Reply:
Sure  👍